software/script/chameleon_cli_unit.py

```diff
@@ -1029,7 +1029,7 @@ def res_value(self, src_blk, src_type, src_key, dst_blk, dst_type, dst_key):
 def _run_mfkey32v2(items):
     output_str = subprocess.run(
         [
-            "mfkey32v2.exe" if sys.platform == "win32" else "./mfkey32v2",
+            ".\\mfkey32v2.exe" if sys.platform == "win32" else "./mfkey32v2",
```

Comment:
Tried following the changes you did and got this to working.
```suggestion
            "./bin/mfkey32v2.exe" if sys.platform == "win32" else "./mfkey32v2",
```

Reply:
Oh, sorry started a new review...

Reply:
That is really weird, I don't understand...
Can you add a `print(default_cwd)` just after the `default_cwd = ` definition (line 41), then run the CLI and report the path displayed? On Linux I get ` /home/user/Github/RfidResearchGroup/ChameleonUltra/software/script/bin`. I expect you'd get something similar under Windows, but based on your suggestion it seems you won't have the `bin` final directory?

Reply:
`C:/Users/USER/Desktop/Chamilion Ultra/CLI/ProxSpace/msys2/home/USER/ChameleonUltra/software/script/bin`
I do have the bin. Could it be I'm running in with the ProxySpace guide?

Reply:
OK thanks a lot! I've tried something else, that should work in any environment. Can you check again?

Reply:
@p-l- This works!

Reply:
Thanks, appreciate your help! Did you notice a performance improvement?

Reply:
Yes, running much faster!